Dockerfile

```
@@ -11,7 +11,8 @@ COPY app/Gemfile.lock /var/task
 COPY app/.bundle /var/task/
 
 # Install gems
-RUN bundle install --path vendor/bundle
+RUN bundle config set path 'vendor/bundle' && \
+    bundle install
```

```suggestion
RUN bundle install
```

そもそも、`vendor/bundle`指定する必要あるのでしょうか？
bundle installだけで問題なければ、`vendor/bundle`は指定しなくていい気がしました。

* [bundle install時に--path vendor/bundleを付ける必要性は本当にあるのか、もう一度よく考えてみよう](https://qiita.com/jnchito/items/99b1dbea1767a5095d85)
* [bundle install時に"[DEPRECATED] The `--path` flag is deprecated"という警告が発生した場合の対応手順](https://qiita.com/jnchito/items/62adbea043abf72fa7cc)

---

私もリンク先のページはみました。
自分も指定なしで問題ない派なのですが、
元々のコードがパスを指定していたので、
なるべく振る舞いが変わらないように修正しました

suggested changeをapplyしてローカル実行したら成功したので
パス指定なしでも問題なしでした。